source/_global.rst

```diff
@@ -24,18 +24,19 @@
 .. |documentation_home| replace:: http://docs.cloudstack.apache.org/
 
 .. Latest version systemvm template name
-.. |sysvm64-name-xen|    replace:: systemvm-xenserver-4.11.3
-.. |sysvm64-name-kvm|    replace:: systemvm-kvm-4.11.3
-.. |sysvm64-name-vmware| replace:: systemvm-vmware-4.11.3
-.. |sysvm64-name-hyperv| replace:: systemvm-hyperv-4.11.3
-.. |sysvm64-name-ovm|    replace:: systemvm-ovm-4.11.3
+
+.. |sysvm64-name-xen|    replace:: systemvm-xenserver-4.13.0
```

Comment:
We don't have new 4.13.0 systemvmtemplate. The 4.11.3.0 systemvmtemplate is actually good enough.

Reply:
@rhtyd I *thought* that the changes that you merged for the system vm necessitate a new template?

Reply:
@PaulAngus a systemvmtemplate is only necessary when changes are done in cloud-early-config script, in master (https://github.com/apache/cloudstack/pull/3126) all the changes from were in other files which would be patched during startup by systemvm.iso and therefore an explicit/new systemvmtemplate is not necessary.

Reply:
there should be changes to the services which were already started/enabled during OS boot up @rhtyd

Reply:
@PaulAngus Yes, this is taken care by bootstrap.sh: https://github.com/apache/cloudstack/blob/master/systemvm/debian/opt/cloud/bin/setup/bootstrap.sh#L67
During patching, first the scripts are patched including bootstrap.sh, which is then executed and will manage/launch services on startup.

Reply:
Let me re-confirm and test if we can do without a new systemvmtemplate.

Reply:
I've changed the system vm templates to 4.11.3 for the time being to get the overall changes in.